.github/CONTRIBUTING.md

```diff
@@ -43,8 +43,7 @@ not already exist.
 ## Submitting changes
 
 * Push your changes to a topic branch in your fork of the repository.
-* Submit a pull request to the repository in the [Aerogear GitHub organization]
-  (https://github.com/aerogear) and choose branch you want to patch 
+* Submit a pull request to the repository in the [Aerogear GitHub organization](https://github.com/aerogear) and choose branch you want to patch 
```

Comment:
we could add the aerogearcatlog org here as well, as both are mentioned on https://aerogear.org/community/#contact